### PR TITLE
[zh-cn] sync cron-jobs.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md [hunk 1]
```diff
@@ -64,17 +64,24 @@ Kubernetes 项目官方并不支持设置如 `CRON_TZ` 或者 `TZ` 等变量。
...
+characters.  This is because the CronJob controller will automatically append
+11 characters to the name you provide and there is a constraint that the
+length of a Job name is no more than 63 characters.
 -->
-为 CronJob 资源创建清单时，请确保所提供的名称是一个合法的
-[DNS 子域名](/zh-cn/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)。
-名称不能超过 52 个字符。
-这是因为 CronJob 控制器将自动在提供的 Job 名称后附加 11 个字符，并且存在一个限制，
-即 Job 名称的最大长度不能超过 63 个字符。
+当控制面为 CronJob 创建新的 Job 和（间接）Pod 时，CronJob 的 `.metadata.name` 是命名这些 Pod 的部分基础。
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md:78]: control plane -> 控制平面

File: content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md [hunk 2]
```diff
@@ -64,17 +64,24 @@ Kubernetes 项目官方并不支持设置如 `CRON_TZ` 或者 `TZ` 等变量。
...
+CronJob 的名称必须是一个合法的
+[DNS 子域](/zh-cn/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)值，
+但这会对 Pod 的主机名产生意外的结果。为获得最佳兼容性，名称应遵循更严格的
+[DNS 标签](/zh-cn/docs/concepts/overview/working-with-objects/names#dns-label-names)规则。
+即使名称是一个 DNS 子域，它也不能超过52个字符。这是因为 CronJob 控制器将自动在提供的 Job 名称后附加 11
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md:83]:
```suggestion
即使名称是一个 DNS 子域，它也不能超过 52 个字符。这是因为 CronJob 控制器将自动在提供的 Job 名称后附加 11
```

File: content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md [hunk 3]
```diff
@@ -64,17 +64,24 @@ Kubernetes 项目官方并不支持设置如 `CRON_TZ` 或者 `TZ` 等变量。
...
+个字符，并且存在一个限制，即 Job 名称的最大长度不能超过 63 个字符。
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md:84]: Please make the sentence more fluent.

It's uncommon to use "并且存在一个限制，即 ...." format.

**Review by my-git9 — COMMENTED** [review]
/lgtm

**Comment by Sea-n** [comment]
Please do this to trigger a re-test while keeping <kbd>lgtm</kbd> label.

```
git reset
git commit --amend --no-edit
git push -f
```

**Comment by gaogao101** [comment]
@Sea-n I have reuploaded it. Thank you for your help.

**Comment by my-git9** [comment]
/lgtm
Thanks!

**Pull request merged**